### ISSUE TITLE
syntax highlighting theme is not changed

### ISSUE THREAD

**Issue by LobanovaVA** [dracula/gitlab#3]
Hi! I followed the instruction on the website https://draculatheme.com/gitlab but I could not see any changes at GitLab Preferences page.

![installed](https://user-images.githubusercontent.com/20883694/192327560-74f70fc2-aa0e-4507-b172-d0a6d5874aaa.png)

![gitlab](https://user-images.githubusercontent.com/20883694/192327556-83c6fb60-823b-4083-b52b-b69bd865147a.png)

**Comment by veejaypee** [comment]
Are you on gitlab.com or on some private gitlab install?

If on private:
Edit the theme in Stylus and add your domain as well (or change gitlab.com to your domain)

**Comment by LobanovaVA** [comment]
Yes, it's private GitLab. Well, the theme appeared in GitLab preference.

But if I choose (in GitLab -/profile/preferences) Dark Mode Color theme and Dracula Syntax highlighting theme together
 
![Screenshot_2023-07-23_14-32-56](https://github.com/dracula/gitlab/assets/20883694/6be39e8b-d682-4350-bc32-84e8e74f5acb)

I get a small color bug.

![photo_2023-07-23_14-18-34](https://github.com/dracula/gitlab/assets/20883694/651c71a2-e16d-4cbe-bc60-75e6d61e7c2d)

**Comment by veejaypee** [comment]
Yes had the same bug, switched back to normal Dark syntax. Expanding this was even more breaking

**Comment by Ayce45** [comment]
FIx on #5
